### PR TITLE
Upgrade to SeaQuery 0.28.0

### PR DESCRIPTION
**Pull request by ikrivosheev**
## PR Info

- Dependencies:
  - https://github.com/SeaQL/sea-schema/pull/90

## New Features

- [x] Upgrade **SeaQuery** to 0.28.0.
- [x] Upgrade **sea-query-binder** to 0.3
- [ ] Upgrade **SeaSchema**

### CODE REVIEW

**Comment by billy1624** [comment]
Sure! Happy to help :)
Let me check

**Comment by billy1624** [comment]
Hey @ikrivosheev, check this https://github.com/ikrivosheev/sea-orm/pull/1

**Comment by ikrivosheev** [comment]
@billy1624 thank you so mach! Well, I am waiting for release sea-schema and the PR is ready)

**Review by billy1624 — COMMENTED** [review]
@ikrivosheev thanks again for the help!!

**Comment by ikrivosheev** [comment]
> @ikrivosheev thanks again for the help!!

You are welcome!

**Review by billy1624 — APPROVED** [review]
Hey @ikrivosheev, happy new year! Thanks for the work!!
I'll merge this into a local branch where I have write permission. Happy vacation :)

**Comment by ikrivosheev** [comment]
> Hey @ikrivosheev, happy new year! Thanks for the work!!
> I'll merge this into a local branch where I have write permission.
> Happy vacation :)

@billy1624 thank you so much! Happy new year)

**Comment by tyt2y3** [comment]
> I'll merge this into a local branch where I have write permission.

We can push WIP branches to the SeaQL repo directly, as I think after the PR is merged it will automatically be deleted?

File: sea-orm-cli/Cargo.toml
Comment (tyt2y3) [inline sea-orm-cli/Cargo.toml:38]: I think sea-schema has been released now?

**Review by tyt2y3 — COMMENTED** [review]
Awesome

**Comment by tyt2y3** [comment]
Does that mean we can now abandon `sea_orm::ColumnDef` and `IdenStatic`?

**Comment by billy1624** [comment]
> Does that mean we can now abandon SeaORM's `ColumnType` and `IdenStatic`?

I think so, we can dump `ColumnType` and `IdenStatic` in SeaORM. However, re-export both of the SeaQuery counterparts in the same SeaORM module to make it backward compatible.

**Comment by billy1624** [comment]
> Does that mean we can now abandon SeaORM's `ColumnType` and `IdenStatic`?

I just started a tracking issue for this https://github.com/SeaQL/sea-orm/issues/1387

**Pull request merged**
